zulip_bots/zulip_bots/provision.py

```diff
@@ -72,7 +74,7 @@ def main():
     # type: () -> None
     current_dir = os.path.dirname(os.path.abspath(__file__))
     bots_dir = os.path.join(current_dir, "bots")
-    bots_subdirs = map(os.path.abspath, glob.glob(bots_dir + '/*'))
+    bots_subdirs = map(lambda d: os.path.abspath(d), glob.glob(bots_dir + '/*'))
```

Comment:
`map` takes a function as the first argument, so the initial version should be fine here.

Reply:
It definitely should, but it results in a bunch of weird errors:
```
Argument 1 to "map" has incompatible type Callable[[AnyStr], AnyStr]; expected Callable[[str], AnyStr]
Argument 1 to "isdir" has incompatible type "AnyStr"; expected "Union[bytes, str, _PathLike[Any]]"
Argument 1 to "parse_args" has incompatible type Iterator[AnyStr]; expected Iterator[str]
```
@neiljp FYI :)